jujugui/static/gui/src/app/components/help/tour/_tour.scss

```diff
@@ -0,0 +1,130 @@
+.tour {
+  $tour-max-width: 850px;
+
+  &.lightbox {
+    @media screen and (min-width: $tour-max-width) {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+    }
+  }
+
+  .lightbox {
+    &__close {
+      @media screen and (max-width: $tour-max-width - 1) {
+        top: .5rem;
+        right: .5rem;
+      }
+    }
+
+    &__wrapper {
+      text-align: left;
+
+      @media screen and (min-width: $tour-max-width) {
+        width: 850px;
+        left: auto;
+        right: auto;
+        top: auto;
+        bottom: auto;
+      }
+
+      @media screen and (max-width: $tour-max-width - 1) {
+        top: 2.5rem;
+        left: 0;
+        right: 0;
+        bottom: 2.5rem;
+      }
+    }
+
+    &__content {
+      padding: 0;
+
+      @media screen and (min-width: $tour-max-width) {
+        height: 530px;
+      }
+    }
+
+    &__slide {
+      &:nth-child(2) {
+        .tour__slide-image {
+          margin-top: 4rem;
+        }
+      }
+      &:nth-child(4) {
+        .tour__slide-image {
+          margin-top: 4rem;
+        }
+      }
+    }
+
+    @media screen and (max-width: $tour-max-width - 1) {
```

Comment:
If we're hiding the button to show the component on small screens, why do we special case them here?

Reply:
The (?) menu disappears on a smaller screen but the GUI help link is shown in the profile menu to enable access on smaller screens, so the component still works.